habitat-lab/habitat/gui/replay_gui_app_renderer.py

```diff
@@ -56,6 +56,7 @@ def __init__(
         cfg.gpu_device_id = 0  # todo
         cfg.force_separate_semantic_scene_graph = False
         cfg.leave_context_with_background_renderer = False
+        cfg.enable_frustum_culling = False  # TODO: Circumvents culling issues
```

Comment:
How about modify this Python code so that it works whether or not the version of habitat-sim supports this flag? In general, it's good to have some backward compatibility with older builds of habitat-sim.